tests/backup/backup_restore_basic_test.go

```diff
@@ -895,3 +897,168 @@ var _ = Describe("{CustomResourceRestore}", func() {
 
 	})
 })
+
+var _ = Describe("{AllNSBackupWithIncludeNewNSOption}", func() {
+	var (
+		contexts                   []*scheduler.Context
+		cloudCredUID               string
+		cloudCredName              string
+		backupLocationName         string
+		backupLocationUID          string
+		backupLocationMap          map[string]string
+		periodicSchedulePolicyName string
+		periodicSchedulePolicyUid  string
+		scheduleName               string
+		appNamespaces              []string
+		backupClusterName          string
+		restoreName                string
+		nextScheduleBackupName     interface{}
+	)
+
+	JustBeforeEach(func() {
+		testrailID := 84760
+		StartTorpedoTest("AllNSBackupWithIncludeNewNSOption", "Verification of schedule backups created with include new namespaces option", nil, testrailID)
```

Comment:
What's the point of assigning testrailID to a var and then using it here? Why can't we use it directly?

Reply:
Convention derived from a previous test case. Nothing else.

Reply:
Cool.

Reply:
Removed testrailID variable.